logging/logging.go

```diff
@@ -17,15 +17,23 @@
 // control where log messages end up.
 package logging
 
-import "log"
+import (
+	"log"
+	"os"
+)
 
 // Verbosef is called to write verbose logs, such as when a new connection is
-// established correctly.
 var Verbosef = log.Printf
 
 // Infof is called to write informational logs, such as when startup has
-// completed.
 var Infof = log.Printf
 
 // Errorf is called to write an error log, such as when a new connection fails.
 var Errorf = log.Printf
+
+// LogDebugToStdout updates Verbose and Info logging to use stdout instead of stderr.
+func LogDebugToStdout() {
+	logger := log.New(os.Stdout, "", log.LstdFlags)
```

Comment:
Not sure if this is best way to do this :thinking:

Reply:
I'm pretty torn on this one too. I don't like that cloud_sql_proxy.go now has two different styles of overriding the logging style.

IMO it would probably be better if there was a "LogVerboseToNowhere" method in this package, and have line 394 of cloud_sql_proxy.go call that instead. Would you mind making that one last change?